qc_schema/dev/properties/mp_properties.py

```diff
@@ -0,0 +1,38 @@
+"""
+The complete list of Moller-Plesset properties.
+"""
+
+mp_properties = {}
+
+mp_properties["mp2_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The unscaled portion of the MP2 correlation energy from same-spin or triplet doubles correlations."
+}
+
+mp_properties["mp2_opposite_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The unscaled portion of the MP2 correlation energy from opposite-spin or singlet doubles correlations."
+}
+
+mp_properties["mp2_single_energy"] = {
+    "type": "number",
+    "description": "The singles portion of the MP2 correlation energy. Zero except in ROHF."
+}
+
+mp_properties["mp2_double_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The doubles portion of the MP2 correlation energy including same-spin and opposite-spin correlations."
+}
+
+mp_properties['mp2_total_correlation_energy'] = {
```

Comment:
Would be useful to also have a total MP energy property. I know it's easy to sum up, but some people like to see the total energy.

Reply:
Sure, missed this.